### PR TITLE
Tasks can run concurrently

### PR DESCRIPTION
**Pull request by ramonsmits**
The scheduler will not prevent tasks to run concurrently.

### CODE REVIEW

File: nservicebus/scheduling/index.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The difference between the following two examples is that in the latter a name i
 - Scheduled tasks, once created, cannot be canceled or changed.
 - While the task repeat interval can be specified, setting a specific time for the task to run is not supported by the scheduler.
 - Each scheduled task will be executed in a new task generated by the Task Parallel Library using `Task.Factory.StartNew(Action)`. This means that by default there will be no transaction scope and that it is up to you to create one if needed.
+- Tasks can be run concurrently if a task its execution duration exceeds the schedule interval. If a task may not overlap then either cancel either the old or new task or wait for the current task to complete.
```
Comment (andreasohlund) [inline nservicebus/scheduling/index.md]: Missing a `during` ?

**Comment by ramonsmits** [comment]
@andreasohlund I don't know what you ment with your comment but I made a change as the sentence was not correct.

File: nservicebus/scheduling/index.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ The difference between the following two examples is that in the latter a name i
 - Scheduled tasks, once created, cannot be canceled or changed.
 - While the task repeat interval can be specified, setting a specific time for the task to run is not supported by the scheduler.
 - Each scheduled task will be executed in a new task generated by the Task Parallel Library using `Task.Factory.StartNew(Action)`. This means that by default there will be no transaction scope and that it is up to you to create one if needed.
+- Tasks can be run concurrently if a task its execution duration exceeds the schedule interval. If task executions may not overlap then either cancel the old or new task or wait for the current task to complete.
```
Comment (SimonCropp) [inline nservicebus/scheduling/index.md]: > wait for the current task to complete

How would one do this?

Comment (SimonCropp) [inline nservicebus/scheduling/index.md]: > Tasks can be run concurrently if a task its execution duration exceeds the schedule interval

this makes no sense

Comment (SimonCropp) [inline nservicebus/scheduling/index.md]: > If task executions may not

??

**Comment by SimonCropp** [comment]
superseded by https://github.com/Particular/docs.particular.net/commit/caeb5d96bbc76570651f87c9d4932dbc77e2be62

**Pull request closed without merging**